gammapy/makers/reduce.py

```diff
@@ -22,14 +22,18 @@ class DatasetsMaker(Maker):
     makers : list of `Maker` objects
         Makers
     stack_datasets : bool
-        Stack into reference dataset or not
+        If True stack into the reference dataset (see `run` method arguments).
     n_jobs : int
-        Number of processes to run in parallel
-        Default is None
-    cutout_mode : str
-        Cutout mode. Default is "partial"
-    cutout_width : str or `~astropy.coordinates.Angle`,
-        Cutout width. Default is None, If Default is determined
+        Number of processes to run in parallel. Default is None.
```

Comment:
default should be 1 no?

Reply:
yes, I also changed the TSMapEstimator for consistency

Reply:
@registerrier No, the default of `None` was introduced on purpose and I think it should remain! The difference is that `n_jobs=None` implements a second path which avoids into calling into `multiprocessing` at all. The reason was that we have seen issues with `multiprocessing` in the past. See e.g. https://github.com/gammapy/gammapy/issues/2453. That's why the default should remain `None` (i.e. not calling into `multiprocessing` at all), optionally users can then use the parallelisation, but it is not guaranteed to work in all situations!

Reply:
running multiprocessing only if n_jobs > 1 is equivalent, cf the change below